lib/src/modules/conditional.dart

```diff
@@ -267,15 +293,27 @@ class Sequential extends _Always {
   /// The input clocks used in this block.
   final List<Logic> _clks = [];
 
+  @override
+
+  /// This sets the Sequential conditions to reset
+  // ignore: overridden_fields
+  Logic? reset;
+
+  @override
+
+  /// This sets a particular Sequential conditions receiver to resetVal
+  // ignore: overridden_fields
+  Map<Logic, Logic> resetVal = {};
+
   /// Constructs a [Sequential] single-triggered by [clk].
   Sequential(Logic clk, List<Conditional> conditionals,
-      {String name = 'sequential'})
-      : this.multi([clk], conditionals, name: name);
+      {Logic? reset, String name = 'sequential'})
```

Comment:
You can use `super.reset` instead of collecting it yourself then passing it to the super constructor.

Reply:
Same for `resetValues`